### ISSUE TITLE
Remove default collections

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/argo#2351]
When a collection is created in argo, the APO for that collection is updated to make the new collection a “default collection” (`//administrativeMetadata/registration/collection`). These nodes are what governs what collections show up when registering items.  Can you tell me if this is functionality we must retain, or could we just show all of the collections for the given APO on the registration page.

See: https://github.com/sul-dlss/argo/blob/main/app/controllers/registrations_controller.rb#L119
https://github.com/sul-dlss/dor-services/blob/1743ea8c144467d0167c333d34cfb62b9d2eb7ec/lib/dor/datastreams/administrative_metadata_ds.rb#L119-L143
https://github.com/sul-dlss/argo/blob/main/app/controllers/collections_controller.rb#L25

**Comment by andrewjbtw** [comment]
How would we show the collections in an APO without using the list in the administrativeMetadata for that APO? I've had to add collections to that datastream to get them to show up in dropdown lists, so I've assumed that adding data there is required.

Last time I checked, if you add a collection to an APO via that collection's Argo page, it won't be added to the administrativeMetadata for the APO and won't appear on the registration dropdown list until you add it there.

I should note also that the Edit APO form also includes the list of collections in an APO and allows you to add or remove collections that way. And I think creating a new collection at the same time as creating an APO will interact with admin metadata as well. So there are other parts of the system that are expecting to interact with that datastream.
